docs/UserGuide.md

```diff
@@ -127,19 +126,27 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-### Deleting a person : `delete`
+### Removing a musician contact: `remove`
 
-Deletes the specified person from the address book.
+Removes a musician contact from the address book.
 
-Format: `delete INDEX`
+**Format:** `remove n/NAME`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+**Examples:**
+* `remove n/Betsy Crowe`
```

Comment:
Feel that using INDEX would be more appropriate here as mentioned in a previous comment.

Reply:
Will make the necessary changes. Thanks!